beprof/crosswise_profile.py

```diff
@@ -0,0 +1,9 @@
+from beprof import profile
+from beprof import curve
+import numpy as np
+
+
+class CrosswiseProfile(profile.Profile):
+    def normalize(self):
```

Comment:
Add a parameter called norm or value with default value 1.0. We may later use it to normalise to 2.0 instead of 1